build.rs

```diff
@@ -238,9 +238,12 @@ fn build_lib(compiled_libraries: &mut HashSet<Libs>, target: &String, lib: Libs)
         .flag_if_supported("-Wno-unused-command-line-argument");
 
     if target.contains("x86_64") || target.contains("i686") {
-        build
-            .flag_if_supported("-msse2") // GNU
-            .flag_if_supported("-arch:SSE2"); // MSVC
+        let target_env = std::env::var("CARGO_CFG_TARGET_ENV").unwrap();
```

Comment:
We should probably use [`is_like_msvc`](https://docs.rs/cc/latest/cc/struct.Tool.html#method.is_like_msvc) method on `build`, as it make sure we use right thing for the compiler (it is possible to set different compiler then the target).

Reply:
Make sense to me. Thanks for pointing that out, @sagudev.